Offseason2022/src/main/java/frc/robot/commands/DriveMotorTest.java

```diff
@@ -4,16 +4,23 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drivetrain;
 
 /**
  *
  */
 public class DriveMotorTest extends CommandBase
 {
-  public DriveMotorTest( )
+  private Drivetrain m_drivetrain;
+  private boolean    m_left;
+
+  public DriveMotorTest(boolean left, Drivetrain drivetrain)
   {
-    // m_subsystem = subsystem;
+    m_drivetrain = drivetrain;
     // addRequirements(m_subsystem);

```

Comment:
Get rid of the commented line for addRequirements--you've put the code in for it two lines lower.